### PR TITLE
Support for PHP 8.1+

### PR DESCRIPTION
**Pull request by mtrop-godaddy**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,13 +12,14 @@
         }
     },
     "require": {
-        "php": "^7.1",
+        "php": "^8.1",
```
Comment (gforsythe-godaddy) [inline composer.json:15]: If you still need 7.x support, I think you can do something like this
```suggestion
        "php": "^7.1|^8.1",
```

**Review by gforsythe-godaddy — APPROVED** [review]
lots of good fixes

File: composer.json (hunk 1 above)
Comment (dsmith4-godaddy) [reply composer.json:15]: We certainly hope not to need PHP 7.x support 🤣

**Review by dsmith4-godaddy — APPROVED** [review]
lgtm

File: composer.json (hunk 1 above)
Comment (mtrop-godaddy) [reply composer.json:15]: Yeah, I think if anybody needs 7.X support they can just use an old release at this point.

**Pull request merged**